### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by cattre**

### CODE REVIEW

File: beers.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    height: 100%;
+    overflow: auto;
+}
+
+.bg {
+    background-image: url('media/bubbles.gif'), url('media/beer_bg.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    /*opacity: 0.5;*/
```
Comment (snack-attack) [inline beers.css:16]: Remove commented out code. Shouldn't go into master!

File: beers.php [hunk 2]
```diff
@@ -1,38 +1,61 @@
...
-$query = $db -> prepare($queryString);
-$query->execute();
-$beers = $query->fetch();
+require_once 'functions.php';
 
+// Create db connection
+$db = connectDB();
+// Query db for beers
+$beers = queryDB($db, $beersQuery);
+// Get section letters
```
Comment (snack-attack) [inline beers.php:9]: Great use of comments!

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ *              Array of query results
+ */
+function queryDB(object $db, string $query) :array {
+    $query = $db->prepare($query);
+    $query->execute();
+
+    return $query->fetchAll();
+}
+
+/**
```
Comment (snack-attack) [inline functions.php:43]: Missing description

File: beers.php [hunk 4]
```diff
@@ -1,38 +1,61 @@
...
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<meta charset="UTF-8">
+		<meta name='viewport' content='width=device-width, initial-scale=1'>
+		<meta charset='UTF-8'>
 	</head>
 	<body>
+        <div class='bg'></div>
 		<header>
-
+            <h1>A world of beer</h1>
```
Comment (snack-attack) [inline beers.php:27]: Can't tell exactly from the diff, but looks like missing indentation here for <h1> tag!

**Review by snack-attack — CHANGES_REQUESTED** [review]
Hey Rich! This looks fantastic, well done! Just a few very minor comments.

Loving your unit tests! 😄

File: beers.css (hunk 1 above)
Comment (cattre) [reply beers.css:16]: Sorry, missed that one!

File: beers.php (hunk 2 above)
Comment (cattre) [reply beers.php:9]: Thanks, I try

File: functions.php (hunk 3 above)
Comment (cattre) [reply functions.php:43]: Sorry, not sure what happened there. Added now.

File: beers.php (hunk 4 above)
Comment (cattre) [reply beers.php:27]: Looks OK to me?

image

**Comment by cattre** [comment]
@snack-attack all done - please review!

File: beers.php (hunk 4 above)
Comment (snack-attack) [reply beers.php:27]: Yep it does indeed, could've been how Github formatted the diff!

**Review by snack-attack — APPROVED** [review]

**Pull request merged**